frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartTileDescription.tsx

```diff
@@ -55,6 +55,7 @@ const QuickStartTileDescription: React.FC<QuickStartTileDescriptionProps> = ({
                 e.preventDefault();
                 e.stopPropagation();
               }}
+              aria-label={t('quickstart~Prerequisites icon')}
```

Comment:
Imagine a screen reader telling you that you've navigated to a button and then it tells you "Prerequisites icon". Do you know what the button does now?

An aria-label is just as important as visible text. Button text should be actionable.

```suggestion
              aria-label={t('quickstart~Show prerequisites')}
```

Don't forget to regenerate the locale files.

Reply:
Thanks for explaining, I've updated the pr